docs/raw/push-notification-server.md

```diff
@@ -0,0 +1,672 @@
+---
+permalink: /spec/16
+parent: Stable specs
+title: 16/PUSH-NOTIFICATION-SERVER
+---
+
+# 16/PUSH-NOTIFICATION-SERVER
+
+> Version: 0.1
+>
+> Status: Raw
+>
+> Authors: Andrea Maria Piana <andreap@status.im>
+
+- [Push Notification Server](#16push-notification-server)
```

Comment:
I wonder if it makes sense to have a basic components section, so it is clear what the pieces are. E.g.

- Central PN server
- PN service
- Alice Device-ID session
- Bob Device-ID session

Can e..g be a graphviz diagram to showcase how registering works, etc.  E.g. https://specs.vac.dev/specs/remote-log.html#flow